tensorflow/compiler/xla/service/llvm_ir/kernel_support_library.cc

```diff
@@ -60,11 +60,14 @@ Status KernelSupportLibrary::IfWithStatus(
     absl::string_view name, llvm::Value* condition,
     const std::function<Status()>& true_block_generator,
     const std::function<Status()>& false_block_generator) {
-  llvm_ir::LlvmIfData if_data = llvm_ir::EmitIfThenElse(condition, name, b_);
+  llvm_ir::LlvmIfData if_data = llvm_ir::EmitIfThenElse(
+      condition, name, b_, false_block_generator != nullptr);
```

Comment:
Insert a comment on what parameter are you populating, e.g. `/*param_name=*/...`

Reply:
done.